packages/go_router/example/lib/named_routes.dart

```diff
@@ -112,7 +117,7 @@ class HomeScreen extends StatelessWidget {
         children: <Widget>[
           for (final String fid in _families.keys)
             ListTile(
-              title: Text(_families[fid]['name']),
+              title: Text(_families[fid]!.name),
```

Comment:
I'd prefer to avoid using the `!` operator. It might be better to iterate over `_families.values` or to use store the value in a local variable and null-check it.

Reply:
fixed